### DISCUSSION TITLE
토글 버튼 컴포넌트

### DISCUSSION THREAD

**Discussion by harryk-im** [prgrms-web-devcourse/Team-Gaerval-Dadok-FE#397] [Ideas]
이슈로 아직 만들지 않은 컴포넌트들이 몇 가지 존재합니다
그중에서 `좋아요 버튼`과 `라디오 버튼`인데요

프로젝트에 버튼들도 많고 어떻게 구분해야 좋을까 고민하다
bool값에 따라 ui가 바뀌는 공통점이 있어
`토글 버튼 컴포넌트`로 이슈를 파서 해당 버튼을 만들어볼까 하는데

여러분들의 생각은 어떠신가요??
더 좋은 의견이 있다면 댓글로 남겨주세요!

<img width="80" alt="image" src="https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/73218463/c0d3859c-bbee-4d73-8876-e1020284032d">

<img width="117" alt="image" src="https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/73218463/973d1fd8-f922-4f98-a8f3-a1f35a325897">

**Comment by minjongbaek** [comment]
좋아요. 일단 토글 버튼 컴포넌트를 새로 만들고 만약 기존의 버튼 컴포넌트에서 제공할 수 있는 방법이나 더 좋은 방법이 있다면 다시 이야기 해봐요.
